core/src/main/java/com/linecorp/armeria/common/websocket/WebSocketWriter.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.websocket;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.common.stream.StreamWriter;
+
+/**
+ * A {@link WebSocket} that you can write {@link WebSocketFrame}s to it.
+ * Use {@link WebSocket#streaming()} to construct.
+ */
+@UnstableApi
+public interface WebSocketWriter extends WebSocket, StreamWriter<WebSocketFrame> {
+
+    /**
+     * Writesa text {@link WebSocketFrame} to this {@link WebSocket}.
```

Comment:
Global comment in this file.
 ```suggestion
     * Writes a text {@link WebSocketFrame} to this {@link WebSocket}.
```

Reply:
😱